learning_resources/constants.py

```diff
@@ -56,6 +64,14 @@ class OfferedBy(Enum):
     scc = "Schwarzman College of Computing"
     ctl = "Center for Transportation & Logistics"
 
+    @staticmethod
+    def values() -> list[str]:
+        return [c.value for c in OfferedBy]
+
+    @staticmethod
+    def names() -> list[str]:
+        return [c.name for c in OfferedBy]
+
```

Comment:
What do you think of this idea:

```python
class OpenEnum(Enum):

    @classmethod
    def values(cls) -> list[str]:
        return [c.value for c in cls]

    @classmethod
    def names(cls) -> list[str]:
        return [c.name for c in cls]


class OfferedBy(OpenEnum):
....

class LearningResourceType(OpenEnum):
...

```

Reply:
What do you think about pulling in the [`named-enum`](https://pypi.org/project/named-enum/) library which provides this and a lot of other nice methods?

Reply:
Looks good to me! Still seems to be actively maintained too.

Reply:
Did this and updated the other enums in `constants.py` to use it too for good measure.